### PR TITLE
Task #46: Read trivia questions from a file

### LINKED ISSUE #46: Read trivia questions from a file
Write code to read the trivia questions from a file, including determining the format of the file, for User Story #9.

Completion Criteria: Pull request that makes questions and answers are read in from a text file and placed in a `QuestionList`, without any questions hard-coded in the Python code.

Due: Tuesday, April 6

### PR DESCRIPTION
**Pull request by sethsarchet**
Created a file to store the trivia questions and answer, then created a way to read them in and store them in the QuestionList class. The format of the trivia_questions_answers.txt file has the question on the first line, then the correspond answer below it.

### CODE REVIEW

File: utils/lists.py [hunk 1]
```diff
@@ -6,7 +6,26 @@
     "You'll be the judge", 'no... (╯°□°）╯︵ ┻━┻', 'no... baka',
     'senpai, pls no ;-;'
 ]
+i = 0
+answers = []
+questions = []
 
+with open ("/Users/sethsarchet/Desktop/QuestionList.txt") as fp:
```
Comment (jluckenbill) [inline utils/lists.py]: This needs changed to a relative path. Pathnames to files should not include user specific names.

File: utils/lists.py [hunk 2]
```diff
@@ -10,6 +10,7 @@
 answers = []
 questions = []
 
+
 with open ("trivia_questions_answers.txt") as fp:
```
Comment (jluckenbill) [inline utils/lists.py:14]: This is good here. This will open the file independent of a specific user.

File: utils/lists.py [hunk 3]
```diff
@@ -6,7 +6,26 @@
...
+    for line in fp:
+        count +=1
+        if count % 2 == 0: # answer
+            answers.append(line)
+            print(answers[a_count])
+            a_count+=1
+        else: # question
+            questions.append(line)
+            print(questions[q_count])
+            q_count+=1
```
Comment (jluckenbill) [inline utils/lists.py]: This section of code is more complex than it needs to be. A simple while loop that checks for EOF as it's terminating condition with two reads in the middle would have been much simpler.

File: utils/lists.py [hunk 4]
```diff
@@ -6,36 +6,28 @@
...
+    q_count = 0
+    
+    for line in fp:
+        count +=1
+        if count % 2 == 0: # answer
+            answers.append(line)
+            a_count+=1
+        else: # question
+            questions.append(line)
+            q_count+=1
```
Comment (dward118) [inline utils/lists.py:26]: This could all be put into a function. It would make the code more readable but it doesn't have to be put into a function.

**Review by jluckenbill — DISMISSED** [review]
I left an individual comment on the code used to read from the file. That comment is only intended as a suggestion, not a requirement. As long as your code works, it's up to you how you want to write it. I'm approving the pull request.

File: utils/lists.py [hunk 5]
```diff
@@ -6,38 +6,29 @@
...
 ]
+i = 0
+answers = []
+questions = []
+
+
+with open ("trivia_questions_answers.txt") as fp:
+    count = 0
+    a_count = 0
+    q_count = 0
```
Comment (NateDawg96) [inline utils/lists.py:17]: All of your code looks good, however I am curious about the purpose of having an a_count and a q_count. Will the question and answer arrays not already have a count or length function that could be used to access this information? Other than that, the code looks great.

Comment (sethsarchet) [reply utils/lists.py:17]: Yeah, that is a very good point. I did not think about using the length function until a few days after submitting the pull request, so I did not know if it was worth going back and changing.

**Review by dward118 — APPROVED** [review]

**Pull request merged**